server/planning/events/events_files.py

```diff
@@ -45,7 +46,16 @@ class EventsFilesService(superdesk.Service):
     def on_create(self, docs):
         for doc in docs:
             # save the media id to retrieve the file later
-            doc['filemeta'] = {'media_id': doc['media']}
+            _file = app.media.get(doc['media'])
+            doc['filemeta'] = {
+                'media_id': doc['media'],
+                'chunk_size': _file.chunk_size,
+                'content_type': _file.content_type,
+                'filename': _file.filename,
+                'length': _file.length,
+                'name': _file.name,
```

Comment:
isn't this same as filename?

Reply:
yes, it is actually an alias for filename. Will drop it.